ocamldoc/ocamldoc.hva

```diff
@@ -20,7 +20,7 @@
 \newcommand\textbar{|}
 \newcommand\textbackslash{\begin{rawhtml}\\end{rawhtml}}
 \newcommand\textasciicircum{\^{}}
-\newcommand\sharp{#}
+\newcommand\hash{#}
```

Comment:
Damien Doligez (2016/05/09 07:04 -0700):
> > @@ -20,7 +20,7 @@
> >  \newcommand\textbar{|}
> >  \newcommand\textbackslash{\begin{rawhtml}\\end{rawhtml}}
> >  \newcommand\textasciicircum{\^{}}
> > -\newcommand\sharp{#}
> > +\newcommand\hash{#}
> 
> Are you sure this is not going to break `ocamldoc`'s or `hevea`'s
> output ?

Regarding hevea, I checked in its source code before submitting the PR
and this macro does not appear in them.

For ocamldoc, I believe that if it was using the macro in its output I
would have found it because I git grepped for it.